pkgs/development/r-modules/wrapper-radian.nix

```diff
@@ -0,0 +1,15 @@
+{ lib, runCommand, R, recommendedPackages, packages, makeWrapper, radian }:
```

Comment:
This file was adapted from the adjacent `wrapper-rstudio.nix`. It is a lot simpler since we don't need to deal with Qt.